sdk/core/core-http-compat/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 1.3.1 (Unreleased)
+## 1.3.1 (2023-01-05)
```

Comment:
maybe wait on the changes in https://github.com/Azure/azure-sdk-for-js/pull/24141/ for this particular SDK?

Reply:
I could also try to port all of the core changes out of that PR today if we want to ship them this month

Reply:
Yeah we can wait. As far as I know no one is waiting for this change to be released, right?

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/24356